### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by MatejMagat305** [opencl-pure/triple-opencl#4] [General]
@johnmichaelboat please in my work dictionary test pass(and when I put go.mod into pure), but I do not know how set up to github ..., should I devide to  3 repo? Or can you look at this?

**Comment by johnmichaelboat** [comment]
Hi MateMagat,
Sorry to not be present. My straight answer is your work dictionary is an
autonomous project. So it should have an independent github repo.
Best Regards,
Johnmichaelboat

Le sam. 16 mars 2024 à 22:16, MatejMagat305 ***@***.***> a
écrit :
